pkg/rules/manager/rule.go

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) The Thanos Authors.
 // Licensed under the Apache License 2.0.
 
-package thanosrule
+package manager
```

Comment:
i wonder if we can name this more concrete. This implementation "manages" rule files, correct? Maybe even `rules/file` makes more sense here then? The struct `file.Manager` could stay, stating that it is "managing" those rule files.

Reply:
maybe but TBH it's mainly a mangers that groups Prom ones.

Reply:
sounds good then

Reply:
actually we change this in next PR ;p so let's merge